src/chart/Bubble.js

```diff
@@ -35,6 +36,23 @@
         return retVal;
     };
 
+    Bubble.prototype.enter = function () {
+        if (this.useTooltip()) {
+            if (!this._parentOverlay && this._parentWidget._parentOverlay) {
+                this._parentOverlay = this._parentWidget._parentOverlay;  //This is because charts do not get an overlay div when they are inside the MultiChart
+            }
+            this._parentOverlay.selectAll(".layout_Tooltip").each(function(d) {
+                d.node().parentNode.remove();
+            });
+            
+            this._tooltip = new Tooltip()
+                .registerWidget(this)
+                .target(this._parentOverlay.node())
+                .render()
+            ;
+        }
+    };
+    
```

Comment:
Agree 100% - I am wondering if it might even be pushed all the way into "widget"?  We could then (re)use it on other charts also (like c3)?  It is my plan that we will use the new Legend with all the charts and the same could be true for this tooltip (as long as the 3rd party ones fire the needed events)?